tests/perf/fair_queue_perf.cc

```diff
@@ -32,19 +32,19 @@
 static constexpr fair_queue::class_id cid = 0;
 
 struct local_fq_and_class {
-    seastar::fair_group fg;
+    seastar::shared_throttle fg;
     seastar::fair_queue fq;
     seastar::fair_queue sfq;
     unsigned executed = 0;
 
-    static fair_group::config fg_config() {
-        fair_group::config cfg;
+    static shared_throttle::config fg_config() {
```

Comment:
Minor: I am just wondering -- maybe the function name could be adjusted too and changed to `throttle_config()`?

Reply:
Okay, please ignore the comment -- I see that at the end of patches we remove it completely.